tasks.py

```diff
@@ -324,12 +373,21 @@ def bandit(context):
     run_command(context, command)
 
 
+@task
+def yamllint(context):
+    """Run yamllint to validate formating adheres to NTC defined YAML standards.
+
+    Args:
+        context (obj): Used to run specific commands
+    """
+    command = "yamllint . --format standard"
+    run_command(context, command)
+
+
 @task
 def check_migrations(context):
     """Check for missing migrations."""
-    command = (
-        "nautobot-server --config=nautobot_secrets_providers/tests/nautobot_config.py makemigrations --dry-run --check"
-    )
+    command = "nautobot-server --config=nautobot/core/tests/nautobot_config.py makemigrations --dry-run --check"
```

Comment:
This *looks* wrong to me...

Reply:
https://github.com/networktocode-llc/cookiecutter-ntc/blob/nautobot-plugin-v1.0/nautobot-plugin/%7B%7B%20cookiecutter.project_slug%20%7D%7D/pyproject.toml#L30C1-L30C1

Should I revert it?

Reply:
Unless I'm overlooking something it doesn't look like CI exercises this invoke command. I suspect if you try it locally with this change it will fail - I *think* it needs to be reverted but would need to test it to be sure.

Reply:
Reverted, will fix it in the templates.